src/main/java/org/sonarlint/intellij/config/global/rules/RulesTreeTable.java

```diff
@@ -100,8 +103,15 @@ public void mouseMoved(final MouseEvent e) {
         if (obj instanceof RulesTreeNode.Rule) {
           var rule = (RulesTreeNode.Rule) obj;
           var label = new JLabel();
-          var severityAndType = rule.severity() + " " + rule.type();
-          label.setText(StringUtil.capitalize(severityAndType.replace('_', ' ').toLowerCase(Locale.US)));
+          // TODO: Remove condition once we have the mapping done on CORE
+          // It should always be new CCT for standalone rules
+          if (rule.attribute() != null && !rule.impacts().isEmpty()) {
+            var highestQualityImpact = Collections.max(rule.impacts().entrySet(), Map.Entry.comparingByValue(Comparator.comparing(Enum::ordinal)));
+            var impactText = StringUtil.capitalize(highestQualityImpact.getValue().toString().toLowerCase(Locale.ENGLISH));
+            var qualityText = highestQualityImpact.getKey().toString().toLowerCase(Locale.ENGLISH);
+            var text = impactText + " " + qualityText;
+            label.setText(StringUtil.capitalize(text.replace('_', ' ').toLowerCase(Locale.US)));
```

Comment:
Is there a reason to use sometimes ENGLISH sometimes US as the locale?

Reply:
Indeed... I don't there is any reason, we have a mix of both in the project. I'd safe it's safe to choose any of them and stick with it... let's choose `ENGLISH`?